__tests__/workers/notifications.ts

```diff
@@ -562,6 +563,50 @@ describe('post added notifications', () => {
   });
 });
 
+describe('post bookmark reminder', () => {
+  it('should add notification for the user that set the reminder', async () => {
```

Comment:
can you add `should be registered` test

Reply:
Right. I think I remember that somewhere. I'll add it now.